docs/ota.md

```diff
@@ -14,6 +14,9 @@ Magisk does not modify most read-only partitions, which means applying official
 
 ### Devices with A/B Partitions
 
+**Notes:** For the moment `Install to Inactive Slot` method won't work for all Pixel 6 and 7 models (https://github.com/topjohnwu/Magisk/issues/5959). Either follow ["Non A/B" Devices](#non-ab-devices) guide below or use a 3rd party tool like [PixelFlasher](https://github.com/badabing2005/PixelFlasher)
```

Comment:
Avoid linking 3rd-party-tools. Even if they are popular/safe atm, it can change at any point.

Reply:
Does saying "use other 3rd party pixel rooting/flashing tool" without explicitly mentioning PixelFlasher or name of other tool work?

Reply:
Aye